assets/src/dashboard/components/layout/provider.js

```diff
@@ -16,23 +16,32 @@
 /**
  * External dependencies
  */
-import { useMemo, useRef, useCallback, useState } from 'react';
+import { useEffect, useMemo, useRef, useCallback, useState } from 'react';
 import { createContext } from '@web-stories-wp/react';
 import PropTypes from 'prop-types';
 
 export const LayoutContext = createContext(null);
 
 const Provider = ({ children }) => {
+  const firstFocusableContentRef = useRef();
   const scrollFrameRef = useRef(null);
   const [telemetryBannerOpen, setTelemetryBannerOpen] = useState(false);
 
+  // Get the first focusable content in the dashboard
+  // so we can send focus there without having to
+  // tab through the whole WordPress dashboard to get back.
+  useEffect(() => {
+    firstFocusableContentRef.current = document
+      .getElementById('web-stories-dashboard')
+      .querySelector(['button', 'a']);
+  }, []);
+
   const scrollToTop = useCallback(() => {
-    const scrollFrameEl = scrollFrameRef.current;
     document.documentElement?.scrollTo?.({
       top: 0,
       behavior: 'smooth',
     });
-    scrollFrameEl?.children[0]?.focus();
+    firstFocusableContentRef.current?.focus();
```

Comment:
wondering if it's better to grab the first focusable child at the time you call the scrollToTop function in case it ever changes throughout the lifecycle of the dashboard? not sure if this ever actually happens tho so might not add benefit 🤷 
```
  useEffect(() => {
    firstFocusableContentRef.current = document
      .getElementById('web-stories-dashboard');
  }, []);

  const scrollToTop = useCallback(() => {
    ...
    firstFocusableContentRef.current?.querySelector(['button', 'a'])?.focus();
    ...
```

Reply:
It only changes when the view is updated (say, from stories to templates) which at that point the useEffect will run and update.  Pretty sure this is a 6 of 12, half a dozen sitch. Happy to update if you feel it reads easier!

Reply:
nope just wanted to verify. makes sense! no need to change